src/guiding.jl

```diff
@@ -73,7 +74,7 @@ function (G::GuidingDriftCache)(u,p,t)
 
   r = P\(ν .- x)
 
-  dl = dot(f(x,p,t) -  ktilde.f(x,ktilde.p,t), r) - 0.5*tr((outer_(g(x,p,t)) - outer_(ktilde.g(x,ktilde.p,t)))*(inv(P) .- r*r'))
+  dl = dot(f(x,p,t) -  ktilde.f(x,ktilde.p,t), r) - 0.5*tr((outer_(g(x,p,t)) - outer_(ktilde.g(x,ktilde.p,t)))*(inv(P) .- outer_(r)))
```

Comment:
@mschauer  do you immediately see which version is correct here? Does this make sense with the current defs of `outer`? Then I could go with `shows` through both functions ..

Reply:
Actually we “own” r and should make sure it’s always of the same size  as the state